WatchFaceFormat/Complications/res/raw/watchface.xml

```diff
@@ -0,0 +1,722 @@
+                    </Font>
+                  </Text>
+                </PartText>
+              </Default>
+            </Condition>
+          </Compare>
+          <!--
+            Image is present, but no text or title, so show the image centrally
+            in the complication, larger then when the text or title is available
+          -->
+          <Compare expression="rangedValue_image_noTextNorTitle">
+            <Condition>
+              <Expressions>
+                <Expression name="monochromatic_ambientImage">
+                  <![CDATA[[COMPLICATION.MONOCHROMATIC_IMAGE] != null && [COMPLICATION.MONOCHROMATIC_IMAGE_AMBIENT] != null]]>
+                </Expression>
+                <Expression name="monochromatic_noAmbientImage">
+                  <![CDATA[[COMPLICATION.MONOCHROMATIC_IMAGE] != null && [COMPLICATION.MONOCHROMATIC_IMAGE_AMBIENT] == null]]>
+                </Expression>
+              </Expressions>
+              <Compare expression="monochromatic_ambientImage">
+                <PartImage x="30" y="30" width="52" height="52" tintColor="[CONFIGURATION.themeColor.1]" alpha="0">
+                  <Variant mode="AMBIENT" target="alpha" value="255" />
+                  <Image resource="[COMPLICATION.MONOCHROMATIC_IMAGE_AMBIENT]" />
+                </PartImage>
+                <PartImage x="30" y="30" width="52" height="52" tintColor="[CONFIGURATION.themeColor.2]">
+                  <Variant mode="AMBIENT" target="alpha" value="0" />
+                  <Image resource="[COMPLICATION.MONOCHROMATIC_IMAGE]" />
+                </PartImage>
+              </Compare>
+              <Compare expression="monochromatic_noAmbientImage">
+                <PartImage x="30" y="30" width="52" height="52" tintColor="[CONFIGURATION.themeColor.1]" alpha="0">
+                  <Variant mode="AMBIENT" target="alpha" value="255" />
+                  <Image resource="[COMPLICATION.MONOCHROMATIC_IMAGE]" />
+                </PartImage>
+                <PartImage x="30" y="30" width="52" height="52" tintColor="[CONFIGURATION.themeColor.2]">
+                  <Variant mode="AMBIENT" target="alpha" value="0" />
+                  <Image resource="[COMPLICATION.MONOCHROMATIC_IMAGE]" />
+                </PartImage>
+              </Compare>
+            </Condition>
+          </Compare>
+        </Condition>
+      </Complication>
+    </ComplicationSlot>
+
+
+    <!--
+      Basic example of a SHORT_TEXT complication.
+
+      Text is always present. In this example, if Icon is also present, this
+      is shown also, and if not, then Title is shown, if present.
+    -->
+    <ComplicationSlot
+        x="56" y="168"
+        width="112" height="112"
+        slotId="1"
+        name="left_complication" displayName="Left"
+        supportedTypes="SHORT_TEXT">
+      <DefaultProviderPolicy
+          primaryProvider="com.google.android.gm/com.google.android.apps.gmail.wear.complications.UnreadEmailsComplicationService"
```

Comment:
Are these considered public API now?

If so, should we document the available ones?

Reply:
Do you mean for the `primaryProvider` or for `defaultSystemProvider`?

For `primaryProvider` (and secondary) you can add any component you like, but there is no guarantee it will be present.

I can add a link to the system providers: https://developer.android.com/reference/android/support/wearable/complications/SystemProviders.html#ANDROID_PAY ?

Reply:
Added further documentation, in the first use of `DefaultProviderPolicy`:

https://github.com/android/wear-os-samples/pull/961/commits/833d83cc50354c92c9e1d3abf2b1502edad3e805

Reply:
I meant the Gmail activity

Reply:
Ah ok. This was just meant to show an example of a complication component name, which deliberately may or may not be available, allowing demonstration of the `defaultSystemProvider`.

I don't think there's any documentation of these, I just picked out some available ones.